src/home_robot_sim/home_robot_sim/env/habitat_ovmm_env/habitat_ovmm_env.py

```diff
@@ -208,7 +208,9 @@ def _preprocess_obs(
             joint=habitat_obs["joint"],
             relative_resting_position=habitat_obs["relative_resting_position"],
             third_person_image=third_person_image,
-            camera_pose=np.asarray(habitat_obs["camera_pose"]),
+            camera_pose=convert_xz_y_to_xyz(
+                opengl_to_opencv(np.asarray(habitat_obs["camera_pose"]))
```

Comment:
@yvsriram I tested it and this works for me. Can you confirm that it doesn't affect the old baselines?

Reply:
@cpaxton

Reply:
Made this configurable and verified that the old heuristic baseline works.